test/cypress/integration/components/filter-blog-posts.js

```diff
@@ -0,0 +1,302 @@
+
+const page = new FilterableListControl();
+const filter = new Filter();
+
+describe( 'Filter Blog Posts based on content', () => {
+  beforeEach( () => {
+    cy.visit( '/about-us/blog/' );
+    // And I open the filterable list control
+    page.openFilterableListControl();
+    // The I click "Show filters" button
+    page.showFilters();
+  } );
+  it( 'Item name search', () => {
+    // When I enter "loan" in the item name input field
+    page.filterItemName( 'loan' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results with the word "loan" in the post title
+    page.results().should( 'contain', 'loan' );
+    // And the page url should contain "title=loan"
+    cy.url().should( 'include', 'title=loan' );
+  } );
+  it( 'Select a single category', () => {
+    // When I select the first checkbox in the Category list
+    filter.checkCategoryName( 'At the CFPB' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results in that category
+    page.notification().should( 'be.visible' );
+    // And the page url should contain "categories=at-the-cfpb"
+    cy.url().should( 'include', 'categories=at-the-cfpb' );
+  } );
+  it( 'Select multiple categories', () => {
+    // When I select all five checkboxes in the Category list
+    filter.checkCategoryName( 'At the CFPB' );
+    filter.checkCategoryId( 'directors-notebook' );
+    filter.checkCategoryId( 'policy_compliance' );
+    filter.checkCategoryId( 'data-research-reports' );
+    filter.checkCategoryName( 'Info for consumers' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results that are in at least one of the selected categories
+    page.notification().should( 'be.visible' );
+    // And the page url should contain "categories=at-the-cfpb"
+    cy.url().should( 'include', 'categories=at-the-cfpb' );
+    // And the page url should contain "categories=directors-notebook"
+    cy.url().should( 'include', 'categories=directors-notebook' );
+    // And the page url should contain "categories=policy_compliance"
+    cy.url().should( 'include', 'categories=policy_compliance' );
+    // And the page url should contain "categories=data-research-reports"
+    cy.url().should( 'include', 'categories=data-research-reports' );
+    // And the page url should contain "categories=info-for-consumers"
+    cy.url().should( 'include', 'categories=info-for-consumers' );
+  } );
+  it( 'Date range to present', () => {
+    // When I enter "01/01/2021" in the From date entry field
+    page.filterFromDate( '2021-01-01' );
+    // And I click "Apply filters" button
+    page.applyFilters();
+    // Then I should see only results dated 01/01/2021 or later
+    page.notification().should( 'be.visible' );
+    page.lastResult().should( 'contain', '2021' );
+    page.lastResult().should( 'not.contain', '2020' );
```

Comment:
I updated date filter tests to use m-meta-header_right.
I plan to update this test to paginate to the last page of results so it should not fail when there are 25+ blog posts in 2022, but this will be hard to verify in 2021 that it will pass in 2022!